vault/core_metrics.go

```diff
@@ -69,8 +70,19 @@ func (c *Core) metricsLoop(stopCh chan struct{}) {
 				continue
 			}
 			if c.perfStandby { // already have lock here, do not re-acquire
-				syncCounter(c)
+				err := syncCounters(c)
```

Comment:
From [CI](https://app.circleci.com/pipelines/github/hashicorp/vault/15613/workflows/39275698-fbdc-49ea-89de-ea876aa43e4d/jobs/158313) this doesn't exist on the OSS side, which is causing an error.